### ISSUE TITLE
Tiny RAPiD on embedded devices

### ISSUE THREAD

**Issue by anderflash** [duanzhiihao/RAPiD#22]
Hi!

Do you plan to provide a tiny version of the network with, for instance, MobileNetv3 as backbone which can be run even on embedded devices?

If not, do you have any hints about how to change parts of the code to make it faster?

Thanks!

**Comment by duanzhiihao** [comment]
Hi, thank you for your interest.

We do have such a plan last year when we were trying to use "YOLOv5" as the detector instead of YOLOv3. But after some experiments, we find that it didn't give a better complexity-accuracy trade-off, so we gave up that idea. The reason might be that our training setting is sub-optimal because we do not have enough GPUs to tune the hyperparameters.

If you would like to customize your own tiny version RAPiD, I suggest replacing the "backbone" and the "Feature Pyramid Network" because they are the most computation-expensive parts:
https://github.com/duanzhiihao/RAPiD/blob/e56ac87b0422d98f2942dbfbc64b745a3a3149ae/models/rapid.py#L27 https://github.com/duanzhiihao/RAPiD/blob/e56ac87b0422d98f2942dbfbc64b745a3a3149ae/models/rapid.py#L54
